cobra/core/model.py

```diff
@@ -418,6 +418,72 @@ def add_reaction(self, reaction):
 
         self.add_reactions([reaction])
 
+    def add_boundary(self, metabolite, type="exchange", reaction_id=None,
+                     lb=None, ub=1000.0):
+        """Add a boundary reaction for a given metabolite.
+
+        There are three different types of pre-defined boundary reactions:
+        exchange, demand, and sink reactions.
+        An exchange reaction is a reversible, imbalanced reaction that adds
+        to or removes an extracellular metabolite from the extracellular
+        compartment.
+        A demand reaction is an irreversible reaction that consumes an
+        intracellular metabolite.
+        A sink is similar to an exchange but specifically for intracellular
+        metabolites.
+
+        If you specify a type other than one of these three, you can
```

Comment:
'If you set the reaction `type` to something else, you must specify the desired identifier of the created reaction along with its upper and lower bound.'